sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -359,6 +380,13 @@ def get_subscription_receiver(self, topic_name, subscription_name, **kwargs):
 
         """
         # pylint: disable=protected-access
+
+        if self._entity_name and topic_name != self._entity_name:
```

Comment:
for sub receiver is it still just topic_name?  and not like topic_name/subscription_name?

Reply:
(from OOB no, topic-only connstr as far as entity-specific goes.)

Reply:
right, connection string only contains the topic name as the entity path